portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase

```diff
@@ -1373,4 +1373,46 @@ definition {
 		ACUtils.viewNameList(nameList = "ac ac");
 	}
 
+	@description = "Bug: LRAC-8233 | Automation ID: LRAC-11537 | Test Summary: Check web behavior shows which data source data came from"
+	@priority = "3"
+	test CheckWebBehaviorSegmentShowsWhichDataSourceDataCameFrom {
+		ACUtils.createBlogsAndAddToPage();
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		var dataSourceName = StringUtil.extractFirst("${propertyName}", "Combined");
+
+		var dataSourceName = StringUtil.trim("${dataSourceName}");
+
+		ACUtils.navigateToSitePage(
+			actionType = "View Blog",
+			documentTitleList = "Blogs AC Title",
+			pageName = "AC Page",
+			siteName = "Site Name");
+
+		ACUtils.closeAllSessionsAndWait();
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
```

Comment:
I think you could change it to use `${assignedPropertyName}`. What do you think?